templates/page.hbs

```diff
@@ -23,6 +23,14 @@
 
     {{{ module footer }}}
 
+    <main>
+      {{#each components}}
+        {{{module this}}}
+      {{/each }}
+    </main>
```

Comment:
remove this extra main element. (maybe was added in the automatic merge resolving git conflicts)